doc/stdenv/stdenv.chapter.md

```diff
@@ -116,18 +116,19 @@ On Linux, `stdenv` also includes the `patchelf` utility.
 
-Dependencies can be broken down along three axes: their host and target platforms relative to the new derivation’s, and whether they are propagated. The platform distinctions are motivated by cross compilation; see <xref linkend="chap-cross" /> for exactly what each platform means. [^footnote-stdenv-ignored-build-platform] But even if one is not cross compiling, the platforms imply whether or not the dependency is needed at run-time or build-time, a concept that makes perfect sense outside of cross compilation. By default, the run-time/build-time distinction is just a hint for mental clarity, but with `strictDeps` set it is mostly enforced even in the native case.
+But even if one is not cross compiling, the platforms imply whether a dependency is needed at run-time or build-time.
 
-The extension of `PATH` with dependencies, alluded to above, proceeds according to the relative platforms alone. The process is carried out only for dependencies whose host platform matches the new derivation’s build platform i.e. dependencies which run on the platform where the new derivation will be built. [^footnote-stdenv-native-dependencies-in-path] For each dependency \<dep\> of those dependencies, `dep/bin`, if present, is added to the `PATH` environment variable.
+The extension of `PATH` is accomplished by simply adding `dep/bin`, if present. Understandibly, this is only done for dependencies who actually also run ("host") on the new derivation's build platform.
 
-The dependency is propagated when it forces some of its other-transitive (non-immediate) downstream dependencies to also take it on as an immediate dependency. Nix itself already takes a package’s transitive dependencies into account, but this propagation ensures nixpkgs-specific infrastructure like setup hooks (mentioned above) also are run as if the propagated dependency.
+While Nix perfectly manages transitive dependencies, a _propagated_ depdency is one that (forward-)inserts itself into transitive downstream packages as _direct_ dependency. Therefore eventual downstream's setup-hooks and tooling also affect it. It is important to note, that propagated dependencies, since they are "fast-forwarded" to the downstream derviation, a _cross_ dependency might actually end up as a _native_ dependency.
```

Comment:
```suggestion
While Nix perfectly manages transitive dependencies, a _propagated_ depdency is one that (forward-)inserts itself into transitive downstream packages as _direct_ dependency. Therefore eventual downstream's setup-hooks and tooling also affect it. It is important to note for propagated dependencies, since they are "fast-forwarded" to the downstream derviation, that a _cross_ dependency might actually end up as a _native_ dependency.
```